src/CHANGES.txt

```diff
@@ -61,6 +61,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Add a PY3-only function for setting up the cachedir that should be less
       prone to races. Add a hack to the PY2 version (from Issue #3351) to
       be less prone to a race in the check for old-style cache.
+    - docbook tool small error in xslt variant
```

Comment:
Change to Fix docbook tool logic when using lxml package? (Always start bullets with Caps please) :)

Reply:
Grrr... I had actually fixed that before pushing up the PR... and here sits the modified CHANGES.txt, uncommitted.  Sorry.